src/QsCompiler/Tests.Compiler/TestCases/LocalVerification.qs

```diff
@@ -1380,8 +1380,6 @@ namespace Microsoft.Quantum.Testing.LocalVerification {
     @Test("QuantumSimulator")
     function ValidTestAttribute10 () : ((Unit)) {}
 
-    function ValidTestAttribute11 (arg : Unit) : Unit { }
```

Comment:
Was this missing the `@Test` attribute before?

Reply:
It was testing local resource estimation only. I removed the attribute, but left the function in my previous PR.